pandas/tests/resample/test_resampler_grouper.py

```diff
@@ -497,3 +501,19 @@ def test_groupby_resample_with_list_of_keys():
         ),
     )
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("keys", [["a"], ["a", "b"]])
+def test_resample_empty_Dataframe(keys):
```

Comment:
Is it possible to combine this with `test_empty` above using `pytest.mark.parametrize`?